.profile

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# this code is executed while initializing the app:
+# https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#profile
+
```

Comment:
This is the place to extract the `S3_BUCKET` and `S3_REGION` env vars from `VCAP_SERVICES` and set them.

Reply:
Unfortunately, the variables would be set for the build, but not for the apache process to use in the running service.  Thus, we need to have them set with cf env.

Reply:
At the point that `.profile` runs, staging has happened and services are bound, but the start command (which fires up httpd) hasn't run yet. So this is the right place to parse `VCAP_SERVICES` and set `S3_BUCKET` and `S3_REGION`; httpd will be able to pick them up.

Reply:
Handled in https://github.com/18F/cf-ex-drupal8/pull/31